### DISCUSSION TITLE
REACT APP ENV VARIABLE EVEN AFTER USING REACT_APP_

### DISCUSSION THREAD

**Discussion by zohaibamir203** [vercel/community#4826] [Help]
### Summary

```
import { initializeApp } from "firebase/app";
import { getAuth } from "firebase/auth";
import { getStorage } from "firebase/storage";

const firebaseConfig = {
  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
  authDomain: process.env.REACT_APP_FIREBASE_AUTH_DOMAIN,
  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
  storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET,
  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID,
  appId: process.env.REACT_APP_FIREBASE_APP_ID,
  measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENT_ID,
};

// Initialize Firebase
const app = initializeApp(firebaseConfig);
const auth = getAuth(app);
const storage = getStorage(app);

export default app;
export { app, auth, storage };

```

This is my code and i have set my env variable on vercel while deploying and they all include prefix REACT_APP_ but it is not working

### Example

https://claircius-frontend.vercel.app/

### Steps to Reproduce

I have shared the link when you try to sign in by any email & password you can see in the Dev Tool network that the API KEY is incorrect. Same API KEY is working in local host so it meant that vercel is not picking env variable for i don't know what reason

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help with debugging?
